zappa/cli.py

```diff
@@ -696,14 +696,14 @@ def deploy(self, source_zip=None):
             if self.manage_roles:
                 try:
                     self.zappa.create_iam_roles()
-                except botocore.client.ClientError:
+                except botocore.client.ClientError as e:
                     raise ClickException(
                         click.style("Failed", fg="red") + " to " + click.style("manage IAM roles", bold=True) + "!\n" +
-                        "You may " + click.style("lack the necessary AWS permissions", bold=True) +
+                        "You may " + click.style("lack the necessary AWS permissions " , bold=True) +
                         " to automatically manage a Zappa execution role.\n" +
                         "To fix this, see here: " +
-                        click.style("https://github.com/Miserlou/Zappa#using-custom-aws-iam-roles-and-policies", bold=True)
-                        + '\n')
+                        click.style("https://github.com/Miserlou/Zappa#using-custom-aws-iam-roles-and-policies", bold=True) + '\n' +
+                        'Exception: ' + str(e) + '\n')
```

Comment:
The exception was never displayed, so there was no good way to figure out how to actually resolve the error.